components/announcement_bar/announcement_bar_controller.tsx

```diff
@@ -13,6 +13,7 @@ import AnnouncementBar from './default_announcement_bar';
 
 import PaymentAnnouncementBar from './payment_announcement_bar';
 import CloudTrialAnnouncementBar from './cloud_trial_announcement_bar';
+import CloudFreemiumTrialEndAnnouncementBar from './freemium_trial_ended_announcement_bar';
```

Comment:
Probably not specific to freemium but general trial ending

```suggestion
import CloudTrialEndAnnouncementBar from './freemium_trial_ended_announcement_bar';
```

Reply:
Updated all occurrences.